workflows/cloudify_system_workflows/deployment_environment.py

```diff
@@ -119,14 +115,24 @@ def create(ctx, **kwargs):
     sequence.add(
         ctx.send_event('Starting deployment policy engine core'),
         ctx.execute_task('riemann_controller.tasks.create',
-                         kwargs=kwargs.get('policy_configuration', {})))
+                         kwargs=policy_configuration or {}))
 
+    return graph
+
+
+@workflow
+def create(ctx, deployment_plugins_to_install, workflow_plugins_to_install,
+           policy_configuration, **_):
+    graph = generate_create_dep_tasks_graph(
+        ctx,
+        deployment_plugins_to_install,
+        workflow_plugins_to_install,
+        policy_configuration)
     return graph.execute()
 
 
 @workflow
 def delete(ctx, **kwargs):
```

Comment:
Hmm it has always been like this, but we can change it, why not. It shouldn't break anything.